src/util/kaldi-table-inl.h

```diff
@@ -1549,6 +1624,8 @@ class RandomAccessTableReaderScriptImpl:
     last_found_ = 0;
     script_.clear();
     current_key_ = "";
+    range_ = "";
+    data_rxfilename_last_ = "";
```

Comment:
You shouldn't use these initializers-- I think they are a C++11 extension.  Anyway strings are automatically empty.
And do you need data_rxfilename_last_?  If so please explain why.